tests/db/model/sample/upsertBulk.js

```diff
@@ -104,7 +104,7 @@ describe('db: sample: upsertBulk: ', () => {
       let successObj;
       for (let i = 0; i < o.length; i++) {
         const s = o[i];
-        if (util.isError(s)) {
+        if (util.isError(s.explanation)) {
           errorCount++;
```

Comment:
Looks like we are checking for success objs as well and there are only 2 objs in request.